### DISCUSSION TITLE
How to update starship to the latest release?

### DISCUSSION THREAD

**Discussion by weeebdev** [starship/starship#1451] [Q&A] [answered]
Is there a way to update starship to the latest release without downloading it manually? I need to keep all my configs and [specific settings](https://github.com/starship/starship/discussions/1410).

**Comment by matchai** [comment, marked as answer]
Depending on how you installed Starship, it should be updated differently:
- If you installed it using the shell script, rerunning the shell script will install the latest version
- If you installed it using a package manager, you can use the package manager's update mechanism to keep Starship up-to-date
